src/pages/Menu/Components/Aside/Aside.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import CategoryList from '../CategoryList/CategoryList';
+
+import './Aside.scss';
+const Aside = () => {
+  const [category, setCategory] = useState([]);
+
+  const fetchData = () => {
+    async function fetchAndSetCategory() {
+      const response = await fetch('/data/categoryData.json');
+      const data = await response.json();
+      setCategory(data);
+    }
+    fetchAndSetCategory();
+  };
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  return (
+    <aside className="leftWrap">
+      <div className="lnb">
+        <div className="lnbInner">
+          <ul className="categoryWarp">
+            {category.map(category => {
```

Comment:
```suggestion
            {category.map({id, categoryName, categoryList} => {
```
경서님! 위와 같이 구조분해할당 사용하시면 가독성이 더 올라가겠네요!

Reply:
네! 수정하였습니다!